qemu/tests/cfg/x86_cpu_flags.cfg

```diff
@@ -35,6 +35,22 @@
                     flags = "arch_capabilities"
                     check_guest_cmd = "cat /sys/devices/system/cpu/vulnerabilities/mds | grep '%s'"
                     expect_items = 'Not affected'
+                - test_la57:
+                    # support RHEL.8 guest or later
+                    no RHEL.6 RHEL.7
+                    type = x86_cpu_flag_la57
```

Comment:
I guess we don't have to create a new py for this case, x86_cpu_flags.py is good enough.

Reply:
Thanks and update.